### ISSUE TITLE
Correct BPTL Concept IDs

### ISSUE THREAD

**Issue by Davinkjohnson** [episphere/connect#510]
@anthonypetersen will add a screenshot of the CIDs in question

@wyattks we'll need to review whether these CIDs are required each time a BOX is created.

**Comment by anthonypetersen** [comment]
![image](https://user-images.githubusercontent.com/18222069/164273436-f0c82e7b-ada7-486e-aae6-cae46e988516.png)

**Comment by brotzmanmj** [comment]
Agreed, and I do not think these need to be default variables in the sense that they are created and exist from the start. I think when we noted them as 'default' in the dictionary, we meant that the value should be set to No unless marked as Yes. @FrogGirl1123 @anthonypetersen Let us know if any questions about that.

**Comment by Davinkjohnson** [comment]
Confirmed all changes have been made as needed.
